src/foam/u2/wizard/debug/TestWizard.js

```diff
@@ -30,6 +30,15 @@ foam.CLASS({
       factory: function () {
         const scenarioMenus = [];
 
+        scenarioMenus.push(this.Menu.create({
+          id: this.id + '/ElementWizardTest',
+          label: 'Element Wizard Test',
+          parent: this.id,
+          handler: foam.nanos.menu.ViewMenu.create({
+            view: { class: 'net.nanopay.cards.test.wizards.ElementTest' }
+          }, this)
+        }));
+
```

Comment:
I'm acknowledging that this is here and it's not easy to move somewhere else. I added NP-7141 to track this separately.